packaging/helm-charts/helm3/strimzi-kafka-operator/templates/70-ClusterRole-strimzi-admin.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.createAggregateRoles -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: strimzi-admin
+  labels:
+    app: {{ template "strimzi.name" . }}
+    chart: {{ template "strimzi.chart" . }}
+    component: entity-operator-role
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    # Add these permissions to the "admin" and "edit" default roles.
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+rules:
+  - apiGroups:
+      - "kafka.strimzi.io"
+    resources:
+      - kafkas
+      - kafkaconnects
+      - kafkaconnects/scale
+      - kafkamirrormakers
+      - kafkamirrormakers/scale
+      - kafkausers
+      - kafkatopics
+      - kafkabridges
+      - kafkabridges/scale
+      - kafkaconnectors
+      - kafkaconnectors/scale
+      - kafkamirrormaker2s
+      - kafkamirrormaker2s/scale
+      - kafkarebalances
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - delete
+      - patch
+      - update
+  - apiGroups:
+      - "core.strimzi.io"
+    resources:
+      - strimzipodsets
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - delete
+      - patch
+      - update
+  {{- end -}}
```

Comment:
Stupid question - should the 2 empty spaces be there? Or should it be aligned with the `if`? (not sure if it actually matters a role in Helm Chart)

Reply:
My bad.. pushed another commit that removes the redundant whitespace.